### ISSUE TITLE
Warning "type RCTView has a shadow set but cannot calculate shadow efficiently"

### ISSUE THREAD

**Issue by zhaomingli007** [SteffeyDev/react-navigation-popover#1]
Hi Steffey,

This plugin helped me a lot. Thank you.

I encountered a warning when run your test from **"react-native-popover-view-test-app"**, each time I click the "settings" icon on top-right.

![image](https://user-images.githubusercontent.com/8056789/47251212-d1b5bf00-d463-11e8-82ec-4f7e184a2100.png)

Any idea about that?

**Comment by SteffeyDev** [comment]
I think it is because I am putting the shadows on an Animated.View.  I don't see this warning when running on my phone, what device are you running on that you see this error with my test app?  If I can reproduce the warning I think I can fix it.

**Comment by zhaomingli007** [comment]
The warning appears on iOS simulator (iPhone with version 11.2) .
I didn't use expo. this warning is from chrome debug console, not a "yellow" box on iPhone simulator.

**Comment by SteffeyDev** [comment]
You mentioned my "react-native-popover-view-test-app" in your original comment, which is an Expo app.  Did you copy-paste the code into a non-expo app and see the warning there?  I can check if I get the same warning in my non-expo app, but it may be a few days.

**Comment by zhaomingli007** [comment]
Yes,  exactly I did copy and paste your test code into my non-expo code.
This is just a warning it does not stuck me.  
Take your time and thank you for your reply.
